scalastyle-config.xml

```diff
@@ -135,9 +135,9 @@ This file is divided into 3 sections:
   <!-- ??? usually shouldn't be checked into the code base. -->
   <check level="error" class="org.scalastyle.scalariform.NotImplementedErrorUsage" enabled="true"></check>
 
-  <!-- As of SPARK-7558, all tests in Spark should extend o.a.s.SparkFunSuite instead of FunSuite directly -->
+  <!-- As of SPARK-7558, all tests in Spark should extend o.a.s.SparkFunSuite instead of AnyFunSuite directly -->
   <check customId="funsuite" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
-    <parameters><parameter name="regex">^FunSuite[A-Za-z]*$</parameter></parameters>
+    <parameters><parameter name="regex">^AnyFunSuite[A-Za-z]*$</parameter></parameters>
```

Comment:
We need to detect both `FunSuite` and `AnyFunSuite` from new your PRs, don't we?

Reply:
`FunSuite` actually doesn't exist anymore:

> For example, org.scalatest.FunSuite was renamed to org.scalatest.funsuite.AnyFunSuite in 3.1.0.


https://www.scalatest.org/release_notes/3.2.0

Reply:
Oh, got it!